public/app/features/logs/components/fieldSelector/ActiveFields.tsx

```diff
@@ -21,31 +21,64 @@ interface Props {
 export const ActiveFields = ({ activeFields, clear, fields, reorder, suggestedFields, toggle }: Props) => {
   const styles = useStyles2(getLogsFieldsStyles);
 
```

Comment:
Why were these changes needed? Also, please remove the comments for every line, which is extremely redundant.